ush/ioda/bufr2ioda/bufr2ioda_adpupa_prepbufr.py

```diff
@@ -243,22 +242,20 @@ def bufr_to_ioda(config, logger):
     logger.info('Creating derived variables - dateTime from hrdr')
 
     cycleTimeSinceEpoch = np.int64(calendar.timegm(time.strptime(reference_time, '%Y-%m-%dT%H:%M:%SZ')))
-    hrdr = Compute_dateTime(cycleTimeSinceEpoch, hrdr)
+    dateTime = Compute_dateTime(cycleTimeSinceEpoch, hrdr)
 
     logger.debug('     Check derived variables type ... ')
-    logger.debug(f'     hrdr      type = {hrdr.dtype}')
+    logger.debug(f'    dateTime      type = {dateTime.dtype}')
 
     end_time = time.time()
     running_time = end_time - start_time
     logger.info(f"Running time for creating derived variables : {running_time} seconds")
 
     # Create the dimensions
-    dims = {
-        'Location': np.arange(0, lat.shape[0])
-    }
+    dims = {'Location': np.arange(0, lat.shape[0])}
 
     # Create IODA ObsSpace
-    iodafile = f"{cycle_type}.t{hh}z.{data_type}.nc"
+    iodafile = f"{cycle_type}.t{hh}z.{data_type}.{data_format}.{cycle}.nc"
```

Comment:
Do we want cycle included in the file name? What are the others doing?

Reply:
@CoryMartin-NOAA thanks, I removed the cycle from the filename.